apps/test_streaming_scenario.cpp

```diff
@@ -186,6 +186,7 @@ void build_incremental_index(const std::string &data_path, const uint32_t L, con
                                                .with_num_frozen_points(num_start_pts)
                                                .build();
 
+    auto index_search_params = diskann::IndexSearchParams(L, insert_threads);
```

Comment:
Why does search_params have an "insert_threads" parameter?

Reply:
Its just reusing the variable. It was like this before so i kept it the same, however if someone wants to pass a different value. they can.